public/cheating.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>You tried to do something suspicious</title>
+  <style type="text/css">
+    body { background-color: #fff; color: #666; text-align: center; font-family: arial, sans-serif; }
+    div.dialog, div.minion {
+      width: 25em;
+      padding: 0 4em;
+      margin: 4em auto 0 auto;
+      border: 1px solid #ccc;
+      border-right-color: #999;
+      border-bottom-color: #999;
+    }
+    div.minion {
+      width: 640px;
+      margin: 0 auto 0 auto;
+      border: none;
+    }
+    h1 { font-size: 100%; color: #f00; line-height: 1.5em; }
+  </style>
+</head>
+
+<body>
+  <!-- This file lives in public/cheating.html -->
+  <div class="dialog">
+    <h1>You tried to do something suspicious</h1>
+    <p>Please go back and try again, this time without trying to cheat the system...we'll be keeping an eye on you!</p>
```

Comment:
Change the error message to indicate why the user is seeing this page (skipping the waiver video).  And the cute error message isn't really necessary, its better for users to use something clear and straightforward.